src/EntityFramework/Core/Common/Utils/MetadataHelper.cs

```diff
@@ -765,6 +765,7 @@ internal static HashAlgorithm CreateMetadataHashAlgorithm(double schemaVersion)
             if (schemaVersion < XmlConstants.EdmVersionForV2)
             {
                 // v1 and v1.1 use old hash to remain compatible
+                // codeql[cs/weak-crypto] MD5CryptoServiceProvider is not used for cryptography/security purposes and we do it only for v1 and v1.1 for compatibility reasons.
```

Comment:
Note: I just copied this same justification from the code analysis suppression above.